### PR TITLE
update ks-install to v3.0.0

### PR DESCRIPTION
**Pull request by pixiake**
Signed-off-by: pixiake <guofeng@yunify.com>

### CODE REVIEW

File: src/test/ks-installer/Chart.yaml [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: v3.0.0
```
Comment (hlwanghl) [inline src/test/ks-installer/Chart.yaml:26]: `appVersion: 3.0.0` per semantic versioning.

Comment (pixiake) [reply src/test/ks-installer/Chart.yaml]: OK, I've modified it. Please review again.

**Pull request merged**